common/static/karma_common.conf.js

```diff
@@ -38,7 +38,7 @@ var libraryFiles = [
     {pattern: 'js/vendor/jquery.truncate.js', included: true},
     {pattern: 'js/vendor/mustache.js', included: true},
     {pattern: 'common/js/vendor/underscore.js', included: true},
-    {pattern: 'js/vendor/underscore.string.min.js', included: true},
```

Comment:
Do you need to also delete the file ``js/vendor/underscore.string.min.js``? If not, how did this work?